src/main/java/org/jenkinsci/plugins/github/config/GitHubServerConfig.java

```diff
@@ -127,6 +162,19 @@ public void setManageHooks(boolean manageHooks) {
     public void setCustomApiUrl(boolean customApiUrl) {
     }
 
+    /**
+     * Gets the optional display name of this server.
+     * @return the optional display name of this server, may be empty or {@code null} but best effort is made to ensure
+     * that it has some meaningful text.
+     */
+    public String getName() {
```

Comment:
I think we can use both - `getDisplayName` and `getName` as `getName` looks like simple getter, but inside contains some logic (which should be avoided for getters).
With displayName we can emphasize that it gets name or some calculated default value